packages/transitive-overlay/src/index.tsx

```diff
@@ -88,14 +89,28 @@ const accessLegFilter = [
 type Props = {
   activeLeg?: Leg;
   transitiveData?: TransitiveData;
+  trimetModeProps: TrimetModeProps;
 };
 
 const TransitiveCanvasOverlay = ({
   activeLeg,
-  transitiveData
+  transitiveData,
+  trimetModeProps
 }: Props): JSX.Element => {
+  const { showRouteArrows = null, ignoreRouteColor = null, trimetTeal = null } =
```

Comment:
instead of making `trimetTeal` a boolean, we could make this more generic and call this `defaultColorOverride` instead